composer.lock

```diff
@@ -366,20 +367,20 @@
         },
         {
             "name": "pear-pear.horde.org/Horde_Nls",
-            "version": "2.1.0",
+            "version": "2.0.3",
```

Comment:
this happens because 2.0.3 is the highest version which allows php7 (from a composer.json perspective)